main.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'nameable'
+require_relative 'app'
+
+def main
+  app = SchoolLibraryApp.new
+  app.main
+end
+
+main
```

Comment:
- [ ] As stated in the project requirement Kindly follow the instruction For the `main.rb` should handle define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:

- Present the user with a list of options to perform.
- Lets users choose an option.
- If needed, ask for parameters for the option.
- Have a way to quit the app. So  Kindly move the above entry method from the` app.rb` into `main.rb`.